src/Core/Content/Product/DataAbstractionLayer/ProductIndexer.php

```diff
@@ -128,7 +128,14 @@ public function update(EntityWrittenContainerEvent $event): ?EntityIndexingMessa
 
         $this->stockUpdater->update($updates, $event->getContext());
 
-        return new ProductIndexingMessage(array_values($updates), null, $event->getContext());
+        $message = new ProductIndexingMessage(array_values($updates), null, $event->getContext());
+
+        $message->addSkip(\array_merge($message->getSkip(), [
+            self::INHERITANCE_UPDATER,
+            self::STOCK_UPDATER,
+        ]);
```

Comment:
```suggestion
        ]));
```

🤔 an idea to make it runnable

Reply:
The only issue I have with editing online :D thank you